HealthcareSystem/PatientWebApp/Controllers/MailController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Backend.Model.Exceptions;
+using Backend.Model.Users;
+using Backend.Service.SendingMail;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PatientWebApp.Validators;
+
+namespace PatientWebApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MailController : ControllerBase
+    {
+        private readonly IMailService _mailService;
+        private readonly PatientValidator _patientValidator;
+
+        public MailController(IMailService mailService)
+        {
+            _mailService = mailService;
+            _patientValidator = new PatientValidator();
+        }
+
+        [HttpPost("welcome")]
+        public async Task<IActionResult> SendWelcomeMail([FromForm] WelcomeRequest request)
+        {
+            try
+            {
+                 _patientValidator.checkIfEmailValid(request.ToEmail);
```

Comment:
Rename checkIfEmailValid to CheckIfEmailValid, which is a convention in our organization. Methods should start with capital letters.

Reply:
Applied in f3dd33a7ec46560896650fb783d63778066c2722.